src/nodes/UsingForDeclaration.js

```diff
@@ -1,10 +1,26 @@
+const {
+  doc: {
+    builders: { line, softline }
+  }
+} = require('prettier');
+
+const printSeparatedList = require('./print-separated-list');
+
 const UsingForDeclaration = {
-  print: ({ node, path, print }) => [
+  print: ({ node, path, print, options }) => [
     'using ',
-    node.libraryName,
+    node.functions && node.functions.length
```

Comment:
I would remove the `.length` check:

```suggestion
    node.functions
```

Even if this is not technically allowed by the grammar, I think it's more future-proof to allow a potential result of `using {} for ...`. The alternative is to use `node.libraryName`, which will be undefined and make the process fail.

Reply:
`node.functions` is always an Array even if it's an empty one :(

Reply:
Oh :man_facepalming: You are right, sorry.